common/src/main/java/com/redhat/cloud/notifications/config/FeatureFlipper.java

```diff
@@ -132,6 +132,9 @@ public class FeatureFlipper {
     @ConfigProperty(name = "notifications.instant-emails.enabled", defaultValue = "false")
     boolean instantEmailsEnabled;
 
+    @ConfigProperty(name = "notifications.remove-user-from-email-template.enabled", defaultValue = "false")
+    boolean removeUserDataFromEmailTemplateEnabled;
+
```

Comment:
This is just out of curiosity. In this PR you're adding support for multi-user single-emails, right? Why is the feature flag called `remove-user-form-email-template`, though :thinking: ?

Reply:
Because is because we can do that because thanks to V2, we can remove user data from email templates.
Bur you are right, this name doesn't reflect the real purpose of this feature, I will update it.